kubeflow/profiles/prototypes/README.md

```diff
@@ -54,14 +54,13 @@ For __stan__ the RoleBinding may look the following
 
 ```yaml
 apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
+kind: ClusterRoleBinding
```

Comment:
I think these changes in the README.md should be paired with code changes in the profiler-controller.

I don't think this will work since an individual will need to have cluster level access, the idea is that each individual's RBAC would be scoped to the namespace.

Reply:
These bindings are not created by user or controller. Cluster admin should create a view clusterrole and cluster rolebindings for profiles resource to even allow the user to create, get profiles. Otherwise user can’t create profiles without rbac. As profile CR is cluster scoped, it has to bind with cluster role and cluster rolebinding.

Reply:
These 2 fixed resources: ClusterRole and ClusterRoleBinding - would users be added to ClusterRoleBinding as subjects?

The Profile CR doesn't need to be cluster scoped, it could be submitted to the kubeflow namespace and the controller would create the new namespace. This would avoid granting the user any cluster level rights.

Reply:
> These 2 fixed resources: ClusterRole and ClusterRoleBinding - would users be added to ClusterRoleBinding as subjects?
> Yes, either groups or service account or User
> The Profile CR doesn't need to be cluster scoped, it could be submitted to the kubeflow namespace and the controller would create the new namespace. This would avoid granting the user any cluster level rights.

Recently, Profile CR is changed to cluster scope from Namespaced [https://github.com/kubeflow/kubeflow/blob/master/kubeflow/profiles/profiles.libsonnet#L16](here) and that's the reason we can't do Role and RoleBindings anymore.

Reply:
thanks for the explanation - This might be a bug or something specific to the golang implementation of the profile_controller. @lluunn do you recall why the Profile CR had to have a scope of cluster?